pennylane/qchem/openfermion_obs.py

```diff
@@ -816,6 +805,7 @@ def decompose(hf_file, mapping="jordan_wigner", core=None, active=None):
 def molecular_hamiltonian(*args, **kwargs):
     """molecular_hamiltonian(molecule, method="dhf", active_electrons=None, active_orbitals=None,\
     mapping="jordan_wigner", outpath=".", wires=None, args=None, load_data=False, convert_tol=1e12)
+
```

Comment:
Why this blank line is added?

Reply:
This doesn't make a change to documentation, I can remove it.